### PR TITLE
fixes #21: adds ability to get and update a property.

### LINKED ISSUE #21: add ability to update/create and view a property for a user
This is linked to the enhancement request in biocache-hubs to support history button.

https://github.com/AtlasOfLivingAustralia/biocache-hubs/issues/102

### PR DESCRIPTION
**Pull request by temi**
This change has some significant impact:
  * It changes value column of table profiles / class UserProperty from VarChar(255) to Text
  * The system using the webservice must be in AuthorisedSystems

### CODE REVIEW

File: grails-app/conf/BootStrap.groovy [hunk 1]
```diff
@@ -8,8 +9,13 @@ class BootStrap {
 
     def init = { servletContext ->
       log.info("Running bootstrap queries")
+
+        def ctx = servletContext.getAttribute(ApplicationAttributes.APPLICATION_CONTEXT)
```
Comment (djtfmartin) [inline grails-app/conf/BootStrap.groovy]: formatting issues here.

File: grails-app/conf/BuildConfig.groovy [hunk 2]
```diff
@@ -43,7 +43,7 @@ grails.project.dependency.resolution = {
     plugins {
         runtime ":hibernate:3.6.10.16"
 
-        runtime ":ala-bootstrap2:2.3-SNAPSHOT"
+        runtime ":ala-bootstrap2:2.4-SNAPSHOT"
```
Comment (djtfmartin) [inline grails-app/conf/BuildConfig.groovy]: Can we use the non-snapshot versions ? is there a change in 2.4-SNAPSHOT that should be released ?

File: grails-app/conf/BootStrap.groovy (hunk 1 above)
Comment (temi) [inline grails-app/conf/BootStrap.groovy]: done

File: grails-app/conf/BuildConfig.groovy (hunk 2 above)
Comment (temi) [inline grails-app/conf/BuildConfig.groovy]: done.
i used 2.4 since 2.3 was giving me a layout exception.

**Pull request merged**